lib/convenient_service/support/dependency_container/entities/method.rb

```diff
@@ -92,20 +102,40 @@ def define_in_module!(mod)
           def ==(other)
             return unless other.instance_of?(self.class)
 
-            return false if full_name != other.full_name
+            return false if slug != other.slug
             return false if scope != other.scope
             return false if body != other.body
+            return false if alias_slug != other.alias_slug
 
             true
           end
 
+          ##
+          # @return [String, Symbol]
+          #
+          def import_name
+            alias_slug || slug
+          end
```

Comment:
This method can be removed.

It is not immediately obvious why a method has `name` and `import_name`. This may confuse end-users.